test/specs/places.js

```diff
@@ -178,14 +210,12 @@ describe('/places', () => {
           assert.equal(body.model, 'place');
           assert.equal(body.action, 'update');
           assert.equal(body.id, '1d6c5bb4-be60-40fd-97a8-b29ffaa2135f');
-          assert.deepEqual(body.data, { ...input, establishmentId: '100' });
+          assert.deepEqual(body.data, { ...input.data, establishmentId: '100' });
         });
     });
 
     it('adds a message to SQS on DELETE', () => {
-      const input = {
-        comments: 'Lorem ipsum dolor'
-      };
+      const input = {};
```

Comment:
we still want to provide comments on DELETE as need to explain why the room is being removed.